pegen/parser_generator.py

```diff
@@ -117,7 +117,7 @@ def name_gather(self, node: Gather) -> str:
         alt = Alt(
             [NamedItem("elem", node.node), NamedItem("seq", NameLeaf(extra_function_name)),],
         )
-        self.todo[name] = Rule(name, None, Rhs([alt]),)
+        self.todo[name] = Rule(name, None, Rhs([alt]), generated=True)
```

Comment:
Manually inspecting the `pegen/parse.c` file produced by running `make test` I found that in `_loop0_107_rule` there's a `PROPAGATE_ERROR` call even though it's generated from this function (I think).